nxrm-ha/templates/statefulset.yaml

```diff
@@ -145,6 +145,12 @@ spec:
             - name: NEXUS_ZERO_DOWNTIME_ENABLED
           {{- $zeroDowntimeEnabled := default false .Values.statefulset.container.env.zeroDowntimeEnabled }}
               value: "{{ toString $zeroDowntimeEnabled }}"
+          {{- if .Values.secret.nexusSecret.enabled }}
+            - name: NEXUS_SECRETS_KEY_ID
+              value: {{ .Values.secret.nexusSecret.nexusSecretsKeyId | quote }}
```

Comment:
We switched to just use the file and included it as a property in the JSON

Reply:
So I should remove this environment variable?

Reply:
Removed the NEXUS_SECRETS_KEY_ID env variable: https://github.com/sonatype/nxrm3-ha-repository/pull/93/commits/c7235ed767e5afd5c238a1249f275401d27ae905

Reply:
@mpiggott